python/ray/setup-dev.py

```diff
@@ -122,6 +122,7 @@ def do_link(package, force=False, skip_list=None, local_path=None):
         skip_list=args.skip,
         local_path="../../../dashboard",
     )
+    do_link("serve", force=args.yes, skip_list=args.skip)
```

Comment:
Could you please do this in a separate PR? Better to keep things orthogonal where possible, makes it easier to track down issues if they happen.

In this case it's pretty minor, but better to be disciplined about it.